docs/troubleshooting/README.md

```diff
@@ -165,35 +171,24 @@ status:
     selected: false
 ```
 
-The solution here is to add the **env:prod** label to the member cluster resource for **kind-cluster-2** as well so that the scheduler can pick the cluster to propagate resources.
-
-### How to verify the latest ClusterSchedulingPolicySnapshot for a CRP?
-
-- We need to have ClusterResourcePlacement's name **{CRPName}**, replace **{CRPName}** in the command below,
-
-```
-kubectl get clusterschedulingpolicysnapshot -l kubernetes-fleet.io/is-latest-snapshot=true,kubernetes-fleet.io/parent-CRP={CRPName}
-```
-
-- Compare ClusterSchedulingPolicySnapshot with the **CRP's** policy to ensure they match (excluding numberOfClusters field from **CRP's** spec)
-- The ClusterSchedulingPolicySnapshot has a label called **number-of-clusters** check to see if it matches the number of clusters requested in **CRP's** **PickN** placement policy.
+#### Resolution:
+The solution here is to add the `env:prod` label to the member cluster resource for `kind-cluster-2` as well, so that the scheduler can select the cluster to propagate resources.
 
 ### How can I debug when my CRP status is ClusterResourcePlacementSynchronized condition status is set to "False"?
 
-**ClusterResourcePlacementSynchronized** condition status is set to **"False"** if the following occurs, the work is not created/updated for a new **ClusterResourceSnapshot**, **ClusterResourceBinding** for a given cluster.
+The **ClusterResourcePlacementSynchronized** condition status is set to **"False"** under the following circumstances: when the work is not created or updated for a new **ClusterResourceSnapshot**.
 
-In the **ClusterResourcePlacement** status section check to see which **placementStatuses** also has WorkSynchronized status set to **false**.
+#### **Investigation Steps:**
 
-From the **placementStatus** we can get the **clusterName** and then check the **fleet-member-{clusterName}** namespace to see if a work objects exists/updated in this case it won't as **WorkSynchronized** has failed.
+- In the **ClusterResourcePlacement** status section, examine the **placementStatuses** to identify clusters with the `WorkSynchronized` status set to **"False"**.
+- Locate the corresponding **ClusterResourceBinding** for the identified cluster. This resource should indicate the status of the `Work`, whether it was created or updated.
+- A common scenario leading to this issue is the user input for the **rollingUpdate** configuration being too strict. Verify the values for `maxUnavailable` and `maxSurge` to ensure they align with your expectations.
 
-We need to find the corresponding **ClusterResourceBinding** for our **ClusterResourcePlacement** which should have the status of **work** create/update.
+#### **Example Scenario:**
```

Comment:
sometimes you're using ** and sometimes you are not. Can we make them consistent?